services/web/server/tests/unit/with_dbs/03/login/test_login_registration.py

```diff
@@ -133,18 +134,20 @@ async def test_registration_with_expired_confirmation(
         autospec=True,
         return_value=LoginSettings(
             LOGIN_REGISTRATION_CONFIRMATION_REQUIRED=True,
-            LOGIN_REGISTRATION_INVITATION_REQUIRED=True,
+            LOGIN_REGISTRATION_INVITATION_REQUIRED=True,  # <----- invitation REQUIRED
```

Comment:
The comment tells me nothing more than what I already can read. Could you maybe change it or remove it?

Reply:
They highlight a the variable that was change with respect to the previous test configuration